data/human/news.txt

```diff
@@ -1227,3 +1336,31 @@ news "graffiti on spaceport wall"
 			"You think piracy is bad and want to stop us? You and what army?"
 			"Messing with the Syndicate, Does That Sound Like Fun to You?"
 			"83 percent of all statistics are just made up."
+			"Ask not what your planet can do for you; ask what you can do for your planet. - based on John F. Kennedy"
```

Comment:
Real people (not to mention, former politicians)?

Reply:
I'll remove the reference. It is indeed unlikely that some graffiti artist would add a proper reference.